src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
Yeah I couldn't see the comment from @ericstj so I'm not sure what he was suggesting. However having the default is only a convenience thing for folks building in VS or the csproj directly and don't want to always be forced to pass the default configuration. It isn't actually required for the general build system.